vmdb/app/views/ops/_rbac_tag_box.html.haml

```diff
@@ -3,7 +3,7 @@
   %table.style1
     %tr
       %td.key
-        = session[:customer_name]
+        = current_brand.customer_name
```

Comment:
Aren't they all small bits when you start? :stuck_out_tongue:

Even so, if feels very un-DRY to have current_brand all over the place, but yeah, I see your points.  Like I said, not 100% sure.  UI code is structured so funny in Rails...probably because Views are not objects. ¯\_(ツ)_/¯